registration/registration-services/src/main/java/io/mosip/registration/service/BaseService.java

```diff
@@ -615,7 +615,6 @@ private List<String> getSubTypes(BiometricType biometricType, String bioAttribut
 					: SingleAnySubtypeType.RIGHT.value());
 			break;
 		case FACE:
-			subtypes.add(BiometricType.FACE.value());

```

Comment:
Change in this method https://github.com/mosip/registration-client/pull/530/files#diff-702cc6e386fed1a995bf370faa1a503e8b67b953c84c56a8799aae3c345ef4a9L326 above is needed because this line is removed from this switch case?
why is this line removed?